pkg/sql/logictest/testdata/logic_test/udf

```diff
@@ -52,6 +52,9 @@ CREATE FUNCTION f() RETURNS INT IMMUTABLE AS $$ SELECT 1 $$;
 statement error pq: no function body specified
 CREATE FUNCTION f() RETURNS INT IMMUTABLE LANGUAGE SQL;
 
+statement error pq: SQL functions cannot have arguments of type record
```

Comment:
minor drive-by nit for the future: including the error code in the test directive is a nice way to enforce we're using error codes correctly. example: https://github.com/cockroachdb/cockroach/blob/0a42bb9f62b98a7bafe1cde43f504636bd2fba2b/pkg/sql/logictest/testdata/logic_test/udf#L25